Task2/dashboard.js

```diff
@@ -0,0 +1,246 @@
+// fetch('https://reqres.in/api/users')
+// .then(res=>res.json())
+// .then(data=>console.log(data))
+
+// let email=localStorage.getItem('email')?localStorage.getItem('email'):''
+// console.log(email);
+// if(email=='')
+// {
+//     alert('u need to login first');
+//     window.location.href="index.html";
+// }
+
+const root = document.getElementById("root");
+const form = document.getElementById("Form");
+const Post = document.getElementById("btn");
+// const list=document.createDocumentFragment();
+const url = "https://reqres.in/api/users";
+let users = new Array();
```

Comment:
We can simply create a new array using array literal instead of using the Array() constructor. Using an array literal provides readability and better compilation speed.

`let users = [ ];`

@Poorvi-3pillar